charts/alfresco-ai-transformer/tests/confimap_test.yaml

```diff
@@ -35,7 +35,7 @@ tests:
         template: config-ai-transformer.yaml
       - equal:
           path: data.FILE_STORE_URL
-          value: http://acs-alfresco-transform-service/alfresco/api/-default-/private/sfs/versions/1/file/
+          value: http://acs-alfresco-transform-service/alfresco/api/-default-/private/sfs/versions/1/file
```

Comment:
test was looking for it because it used to be like that since forever

checking https://github.com/Alfresco/alfresco-helm-charts/pull/203 looks good

Reply:
👍 also tf pipeline is already proving cm values without trailing slashes